### PR TITLE
Add release notes for November release

### PR DESCRIPTION
**Pull request by markcowl**

### CODE REVIEW

File: docs/release-notes/release-2023-11-07.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+:::
+
+## New Features
+
+### `@typespec/bundler` New Features
+
+- Add a new cli for `typespec-bundler`
+- Expose a bundle manifest with the relative importmap, package name and version.
+- Add new hook in emitter famework for handling circular references.
+- `TypeEmitter` now supports Tuples.
```
Comment (timotheeguerin) [inline docs/release-notes/release-2023-11-07.md:18]:
```suggestion
- Emitter framework: `TypeEmitter` now supports Tuples.
```

Comment (timotheeguerin) [reply docs/release-notes/release-2023-11-07.md:18]: it also should be under compiler

Comment (timotheeguerin) [reply docs/release-notes/release-2023-11-07.md:18]: a few of those are actually

File: docs/release-notes/release-2023-11-07.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+# Release Notes November 2023 (2023-11-07)
+
+:::danger
+This release contains **breaking changes**
+:::
+
+## New Features
+
+### `@typespec/bundler` New Features
```
Comment (timotheeguerin) [inline docs/release-notes/release-2023-11-07.md:13]: I would maybe move that one lower as this is kinda an tool only relevant if you want ot make your own playground

**Review by timotheeguerin — APPROVED** [review]

File: docs/release-notes/release-2023-11-07.md (hunk 1 above)
Comment (markcowl) [reply docs/release-notes/release-2023-11-07.md]: Yeah, found those

**Pull request merged**